src/patharray.js

```diff
@@ -100,6 +100,63 @@ SVG.extend(SVG.PathArray, {
 
     return this
   }
+  // Test if the passed path array use the same commands as this path array
+, haveSameCommands: function(pathArray) {
```

Comment:
Don't worry, it's nothing that messy! This method just compares 2 path array to see if they use the same path data commands (commands like: M, C, A, etc...). Looking back at the name of this method, I can see how it can be misleading. Maybe I should change it to something else like: _useSameCommands_, _useSamePathCommands_, _useSamePathDataCommands_ or something else, I'm open to suggestion.